paasta_tools/tron/client.py

```diff
@@ -109,3 +109,17 @@ def list_namespaces(self):
         """Gets the namespaces that are currently configured."""
         response = self._get('/api')
         return response.get('namespaces', [])
+
+    def get_job_content(self, job):
+        return self._get(f"/api/jobs/{job}/")
+
+    def get_latest_job_run_id(self, job_content):
```

Comment:
This isn't a super efficient way to do this.

The only reason we do this in check_tron_jobs is because we need to collect all the runs for context.

For here, if you only want the most recent run, just do `num_runs=1`

Reply:
`?num_runs=1` on the `get_job` function

Reply:
Eh disregard. My approach will fetch "scheduled" jobs, and really we want the most latest "completed" job.